### PR TITLE
Model Submission for Coordinated Robotics Rocky Sensor Configs 1, 2 and 3

### PR DESCRIPTION
**Pull request by knoedler**
This is the model submission for Coordinated Robotics Rocky.  It is based on a rock crawler RC vehicle.  Sensor config 1 is the base configuration.  Sensor config 2 adds breadcrumbs.  Sensor config 3 has a 2d lidar instead of breadcrumbs since there is not space for both on the physical robot.

### CODE REVIEW

File: submitted_models/coro_rocky_sensor_config_3/model.sdf [hunk 1]
```diff
@@ -0,0 +1,945 @@
...
+            <stddev>0.007</stddev>
+          </noise>
+        </camera>
+      </sensor>
+
+      <sensor name="camera_rear" type="camera">
+        <pose>0.069 -0.03 0.235 0 0 3.141592653589793</pose>
+        <always_on>1</always_on>
+        <update_rate>30</update_rate>
+        <camera name="camera_front">
```
Comment (ibrahimkakbar) [inline submitted_models/coro_rocky_sensor_config_3/model.sdf:441]: <camera name="camera_rear">

Comment (knoedler) [reply submitted_models/coro_rocky_sensor_config_3/model.sdf]: Thank you.  I have fixed the camera name.

File: submitted_models/coro_rocky_sensor_config_1/model.sdf [hunk 2]
```diff
@@ -0,0 +1,908 @@
...
+
+      
+      <!-- Real robot has 2 x B0203 with LN008 each with a 180 deg hor x 130 deg vert FOV
+           Since fisheye cameras are not modeled, wide HD cameras are used in the model -->
+      <sensor name="camera_front" type="camera">
+        <pose>0.111 0.03 0.235 0 0 0</pose>
+        <always_on>1</always_on>
+        <update_rate>30</update_rate>
+        <camera name="camera_front">
+          <horizontal_fov>2.4</horizontal_fov>
```
Comment (acschang) [inline submitted_models/coro_rocky_sensor_config_1/model.sdf:399]: Please adjust the modeling of the LN008 lens field of view to match the previously accepted standard of 2 cameras with 95 degree field for each camera.

Comment (knoedler) [reply submitted_models/coro_rocky_sensor_config_1/model.sdf:399]: I will do that.  For the visual, should I show four cameras or should it match the robot with two?

Comment (acschang) [reply submitted_models/coro_rocky_sensor_config_1/model.sdf:399]: Follow the precedent set with your other models.  I'll check now to confirm which precedent you set.

Comment (knoedler) [reply submitted_models/coro_rocky_sensor_config_1/model.sdf]: Cameras updated to follow precedent.

**Review by acschang — APPROVED** [review]
The initial assessment of the model is complete and no changes are currently warranted. This submission will continue to be evaluated in the next stage of our review process.

**Review by mjcarroll — APPROVED** [review]

**Comment by nkoenig** [comment]
Size[0.573261 0.356562 0.40162]

**Pull request merged**